packages/@sanity/cli/src/actions/init-project/initProject.ts

```diff
@@ -792,7 +792,7 @@ export default async function initSanity(
         frameworkList: frameworks as readonly Framework[],
       })
 
-      if (framework && framework.slug !== 'sanity') {
+      if (framework && framework.slug !== 'sanity' && framework.envPrefix) {
```

Comment:
We could have conditionally rendered the text `, using prefix...` and showed `Detected framework` regardless, but I think there's little value to showing a detected framework if nothing _really_ happens. I think we should show it once we add new increments that depend on the framework of choice.

Reply:
| Remix app | Vue (using Vite) |
|--------|--------|
| <img width="500" alt="image" src="https://user-images.githubusercontent.com/16625915/226095504-bae5ce83-ee61-4043-9b17-d5c6067fba88.png"> | <img width="450" alt="image" src="https://user-images.githubusercontent.com/16625915/226095522-2c5ffa58-2b5d-429e-b244-948e34fb6319.png">